src/tools/w3dviewer/graphicview.cpp

```diff
@@ -236,7 +236,7 @@ void CGraphicView::OnMouseMove(UINT nFlags, CPoint point)
 
         v.X = (((float)point.x - (float)(r.right >> 1)) / (float)(r.right >> 1)
                   - ((float)m_mousePos.x - (float)(r.right >> 1)) / (float)(r.right >> 1))
-            * m_radius * -1.0f;
+            * m_radius;

```

Comment:
The * -1.0f does exist in the original binary.

Reply:
I can absolutely confirm that this `* -1.0f` causes inverted movement (camera moves left when moving right).

Reply:
I tested the latest w3dview from the Thyme trunk alongside the original Westwood w3dview and when I hold down both mouse buttons and move left and right it behaves the same. But if I use the w3dview from this PR, it behaves differently.
That means the left and right movement currently in Thyme is correct to the original as far as I can tell.

Reply:
Ok. So the intent was to have inverted movement on both axis. Ok then I will test your proposal.

Reply:
Tested and worked. Both axis are inverted now.